src/App.tsx

```diff
@@ -2,108 +2,24 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { PreparedPost } from './types/PreparedPost';
+
+const preparedPost: PreparedPost[] = postsFromServer.map(
```

Comment:
Consider renaming the variable 'preparedPost' to 'preparedPosts' to better represent multiple posts 📚